src/utils/riskUtils.js

```diff
@@ -1,10 +1,12 @@
 import handlebars from 'handlebars'
 import RecaptchaV2Invisible from "./risk-providers/recaptcha-v2-invisible";
 import RecaptchaV3 from "./risk-providers/recaptcha-v3";
+import RecaptchaEnterprise from "./risk-providers/recaptcha-v3";
```

Comment:
Should be:

import RecaptchaEnterprise from "./risk-providers/recaptcha-enterprise";

Reply:
Fixed.